### PR TITLE
Stagger and randomize

### PR DESCRIPTION
**Pull request by andrewzhang707**
I edited the DodgyDot class so that it now includes a variable "useTargeting", which randomly chooses whether or not the DodgySquare will try to target the player's face. Now some squares will target the player's face while others will randomly appear on the screen. The squares will also now be staggered in timing.

### CODE REVIEW

File: facerecognition.py [hunk 1]
```diff
@@ -118,6 +119,7 @@ def Game():
                 )
             else:
                 dot.MoveRandomly(refreshTime)
+                print("Random")
```
Comment (prcHeavyz) [inline facerecognition.py]: Let's try not to have debug code on main.

File: facerecognition.py [hunk 2]
```diff
@@ -14,6 +14,7 @@ def __init__(self, height, width):
         self.y = random.randrange(0, self.height, 1)
         self.color = (0, 255, 0)
         self.danger = False
+        self.useTargeting = random.randrange(1,3,1)
```
Comment (prcHeavyz) [inline facerecognition.py]: What do you think about making this variable a boolean instead of an integer?

**Review by prcHeavyz — CHANGES_REQUESTED** [review]
Looks good for initial pass, what are your thoughts on the proposed improvements?

File: facerecognition.py (hunk 2 above)
Comment (andrewzhang707) [reply facerecognition.py]: Nice catch! Thanks will do.

File: facerecognition.py (hunk 1 above)
Comment (andrewzhang707) [reply facerecognition.py]: Sounds good!

Comment (prcHeavyz) [reply facerecognition.py]: It do be gone tho

**Review by prcHeavyz — APPROVED** [review]
Looks good

**Review by alexzhang7 — APPROVED** [review]
Changes look good.

**Pull request merged**